client/main.lua

```diff
@@ -29,13 +29,13 @@ end
 
 local function confirmationCheck()
     local alert = lib.alertDialog({
-        header = 'Wait a minute!',
-        content = 'Are you sure you wish to proceed?',
+        header = 'Espere um minuto!',
```

Comment:
These are not language files. The default language for prompts is English.
If you want to translate this messages, create keys in locales/en.json and locales/es.json.